### DISCUSSION TITLE
[bug?] [ios] notifications for discussion comments

### DISCUSSION THREAD

**Discussion by VuiMuich** [community/community#4042] [Mobile] [answered]
When following a notification for a new comment in a discussion the iOS App just opens the discussion, but fails to jump to the exact comment. In long discussions heavily forked into sub-threads it is quite annoying to find the reply of concern. 
On the contrary the desktop browser version (tbh I have not checked how mobile Safari is doing) is working just fine: jumping from notifications right to the comment and highlighting it.
I am not fully sure if this is a bug or actually just not implemented yet, but I miss this dearly!

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to link to comments internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!
